NDB.Covid19/NDB.Covid19.Droid/Views/AuthenticationFlow/CountriesConsentActivity.cs

```diff
@@ -26,7 +26,8 @@ public class CountriesConsentActivity: AppCompatActivity
         private TextView _notificationText;
         private TextView _beAwareText;
         private TextView _consentExplanationText;
-        private Button _consentButton;
+        private Button _consentWithEUButton;
+        private Button _consentWithoutEUButton;
```

Comment:
Change id to "consentButton_onlyNorway"

Reply:
Went with _consentButtonOnlyNorway to follow convention